tox.ini

```diff
@@ -107,5 +96,5 @@ exclude_lines = if __name__ == ["']__main__["']:
 addopts = -rsxX
 rsyncdirs = tests tox
 looponfailroots = tox tests
-norecursedirs = .hg .tox
+norecursedirs = .hg {toxworkdir}
```

Comment:
is this working as intended? I kind of doubt that pytest is going to properly substitute `toxworkdir`